lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -49,7 +49,7 @@ class LowerXMRPass : public LowerXMRBase<LowerXMRPass> {
 
   void runOnOperation() override {
     dataFlowClasses = llvm::EquivalenceClasses<Value, ValueComparator>();
-    auto &instanceGraph = getAnalysis<InstanceGraph>();
+    hw::InstanceGraphBase &instanceGraph = getAnalysis<InstanceGraph>();
```

Comment:
I think we should avoid using the InstanceGraphBase type. Could we stick to auto or `firrtl::InstanceGraph` instead, or just `InstanceGraph` if FIRRTL was imported, but `hw` wasn't?